runtime/src/iree/tooling/CMakeLists.txt

```diff
@@ -165,6 +166,29 @@ iree_cc_test(
     "requires-filesystem"
 )
 
+iree_cc_library(
+  NAME
+    parameter_util
+  HDRS
+    "parameter_util.h"
+  SRCS
+    "parameter_util.c"
+  DEPS
+    iree::base
+    iree::base::internal::file_io
+    iree::base::internal::flags
+    iree::base::internal::path
+    iree::hal
+    iree::io::formats::gguf
+    iree::io::formats::safetensors
```

Comment:
These formats aren't always included in the runtime, right? Looks like just added as deps to tools and then users can include if they want them?

Reply:
correct - base runtime doesn't pull in the formats - not even the iree/io/ tree or the parameters module - so bare metal and such don't need any of this